README.md

```diff
@@ -12,7 +12,8 @@ Currently, two official plugins are available:
 for the final my additional page for submission will be the "about" page.
 
 Project domain - "https//MrWatson13/Loli-B-Bakery.github.io"
-Backend domain - https://github.com/MrWatson0531/loli-b-back2.git
-live site domain - "https//:loli-b-bakery.smelly.cc"
+Backend domain - [Backend](https://github.com/MrWatson0531/loli-b-back2.git)
+
```

Comment:
* Please, remove the `Escape` listener correctly https://snipboard.io/LxZXru.jpg . 

It should be like this (inside the `if` construction):

```jsx

  useEffect(() => {
    //if a modal is open, then add the keydown event listener
    if (activeModal !== "") {
      const handleEscapeClose = (e) => {
        if (e.key === "Escape") {
          closeActiveModal();
        }
      };
      document.addEventListener("keydown", handleEscapeClose);
      // cleanup function
      return () => {
        //remove the keydown event listener
        document.removeEventListener("keydown", handleEscapeClose);
      };
    }
  }, [activeModal, closeActiveModal]);
```